src/pages/auth/state/emailPromptAtom.ts

```diff
@@ -0,0 +1,3 @@
+import { atom } from 'jotai'
+
+export const isEmailPromptModalOpenAtom = atom<boolean>(false)
```

Comment:
nice, your first atom eh? :D

Reply:
Yeah it is my first atom :D

Reply:
I'd say this is good spot for the rest of the atom.